sandbox/grommet-app/src/components/GlobalHeader/GlobalHeader.jsx

```diff
@@ -37,90 +36,62 @@ export const GlobalHeader = ({
   activeTheme,
   backgroundBack,
   setBackgroundBack,
-  workspace,
-  setWorkspace,
 }) => {
   const theme = useContext(ThemeContext);
-  const [open, setOpen] = useState(false);
-  const boxProps = useMemo(
-    () =>
-      workspace === 'Acme Next'
-        ? {
-            elevation: 'small',
-            round: {
-              corner: 'bottom',
-              size: 'medium',
-            },
-          }
-        : {
-            border: { color: 'border-weak', side: 'bottom' },
-          },
-    [workspace],
-  );
-
+  const breakpoint = useContext(ResponsiveContext);
   return (
-    <Page
-      // kind={window.location.pathname === '/' ? 'full' : 'wide'}
-      kind="full"
-    >
+    <Page kind="full">
       <PageContent pad="none">
         <Box
           direction="row"
           justify="between"
           align="center"
           background={
-            activeTheme === 'refresh'
-              ? { color: 'background-primary-default', dark: true }
+            activeTheme === 'next'
+              ? 'background-primary-xstrong'
               : 'background-front'
           }
+          elevation={activeTheme === 'next' ? 'none' : 'small'}
           pad={{ horizontal: 'xsmall', vertical: 'small' }}
-          {...boxProps}
         >
           <Box direction="row" gap="medium" flex>
             <Box direction="row" gap="xsmall">
               <Button icon={<Menu />} />
               <Box border={{ side: 'left', color: 'border-weak' }} />
             </Box>
-            <Box height="32px" width="90px" align="start">
-              <Image
-                src={`/hpe_greenlake_grn_${
-                  activeTheme === 'refresh' ? 'rev' : 'pos'
-                }_rgb.svg`}
-                fit="contain"
-              />
-            </Box>
-            <Box width="large">
-              <TextInput
-                icon={<Search />}
-                placeholder="Type / to search assets, docs, serices, hardware, and more"
-              />
-            </Box>
+            <Link to="/">
+              <Box height="32px" width="90px" align="start">
+                <Image
+                  src={`/hpe_greenlake_grn_${
+                    activeTheme === 'current' && !theme.dark ? 'pos' : 'rev'
+                  }_rgb.svg`}
+                  fit="contain"
+                  alt="HPE GreenLake badge"
+                />
+              </Box>
+            </Link>
+            {!['xsmall', 'small'].includes(breakpoint) ? (
+              <Box width={breakpoint === 'medium' ? 'medium' : 'large'}>
+                <TextInput
+                  icon={<Search id="global-header-search" />}
+                  placeholder="Type / to search assets, services, and more"
+                  aria-labelledby="global-header-search"
+                />
+              </Box>
+            ) : undefined}
```

Comment:
I'm sure this behavior aligns with current header specs, but for consideration in future iterations I'd expect search to collapse into an icon only button which folds into the rest of the menu items in the header.